R/run_episode_file.R

```diff
@@ -326,3 +326,65 @@ join_cohort_lookups <- function(ep_file_data, year) {
 
   return(join_cohort_lookups)
 }
+
+#' Join SPARRA and HHG
+#'
+#' @inheritParams store_ep_file_vars
+#'
+#' @return The data including the SPARRA and HHG variables matched
+#' on to the episode file.
```

Comment:
This should be in it's own script as it will also be used in the indiv file

Reply:
Done